apps/web/src/components/AssessmentStepTracker/utils.ts

```diff
@@ -269,3 +270,30 @@ export const filterResults = (
     };
   });
 };
+
+const consideredDisqualifiedStatuses = [
+  PoolCandidateStatus.ScreenedOutApplication,
+  PoolCandidateStatus.ScreenedOutAssessment,
+  PoolCandidateStatus.ScreenedOutNotInterested,
+  PoolCandidateStatus.ScreenedOutNotResponsive,
+];
+
+const isConsideredDisqualifiedStatus = (
```

Comment:
Instead of this, I recommend using the functions that already exist in utils/poolCandidate.ts. Basically, replace this is with `isDisqualifiedStatus(candidate.status) || isRemovedStatus(candidate.status)`

Reply:
That is a better idea, instead of doing what I did in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9801/commits/814212bbec0fef1a4901636b7f0e7604cff98f46, I should have done that. Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9801/commits/f1cb5046b19f8aa222e451f4e8bf9d679f4ff71d